packages/store-ui/src/atoms/Price/Price.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import type { Meta, Story } from '@storybook/react'
```

Comment:
One small nit. Can you just write it as mdx? An example can be found at:
https://github.com/vtex/faststore/pull/759

Reply:
No, just ignore what I said. I saw the wrong thing